examples/webgl_multiple_elements_text.html

```diff
@@ -219,7 +219,10 @@
 
 			}
 
+			let lastFrame = performance.now();
 			function render() {
+				const delta = (performance.now() - lastFrame) / 1000;
```

Comment:
Please use `THREE.Clock` for computing the time delta value.

Reply:
Ooh, neat! It's switched over now.